.changes/unreleased/BUG FIXES-20231006-163235.yaml

```diff
@@ -0,0 +1,6 @@
+kind: BUG FIXES
+body: 'internal/fwserver: Fix `Data Source Type Not Found` errors when `GetProviderSchema`
```

Comment:
It might be good to call this out from the perspective of when the problem can occur and the exported "providerserver" package can act as a catch-all category for internal provider server bits.

```markdown
providerserver: Prevented `Data Source Type Not Found` and `Resource Type Not Found` errors with Terraform 1.6 and later
```

Reply:
That message sounds good to me 👍🏻 - thanks for the suggestion